src/Nethermind/Nethermind.Synchronization/SnapSync/SnapProvider.cs

```diff
@@ -29,6 +30,9 @@ public class SnapProvider : ISnapProvider
 
         private readonly ProgressTracker _progressTracker;
 
+        // This is actually close to 97% effective.
+        private readonly LruKeyCache<ValueHash256> _codeExistKeyCache = new(1024 * 16, "");
```

Comment:
512kb of Keccaks; does/can this get dumped after sync?

Reply:
I think we can drop/clean it after the sync finishes.